src/fontra/client/web-components/plugin-manager.js

```diff
@@ -0,0 +1,158 @@
+import { ObservableController } from "../core/observable-object.js";
+import * as html from "../core/unlit.js";
+import { SimpleElement, createDomElement } from "../core/unlit.js";
+import "/web-components/add-remove-buttons.js";
+import { dialog, dialogSetup } from "/web-components/modal-dialog.js";
+import { UIList } from "/web-components/ui-list.js";
+
+export class PluginManager extends SimpleElement {
+  static styles = `
+  .plugin-manager {
+    margin: 1rem;
+    display: grid;
+    grid-auto-rows: auto auto;
+    grid-gap: .5rem;
+  }
+  .buttons {
+    display: flex;
+    gap: 0.2rem;
+  }
+
+  .no-plugins {
+    padding: 0.4rem 0.2rem;
+  }
+  `;
+  constructor() {
+    super();
+    this.contentElement = this.shadowRoot.appendChild(html.div());
+    this.contentElement.classList.add("plugin-manager");
+    const observable = new ObservableController({
+      plugins: [],
+    });
+    observable.synchronizeWithLocalStorage("fontra.plugins");
+    this.observable = observable;
+    this.pluginList = new UIList();
```

Comment:
Adding
```js
    this.pluginList.minHeight = "5em";
```
after this line makes the list not disappear when it is empty. I think that's better in this case.

Perhaps we should add a simple header "Fontra plugins:" just before the list.

I think you could then get rid of the "No plugins added." field.

Reply:
Ok. I did. "No plugins added" deleted. Looks better.